pkg/destroy/azure/azure.go

```diff
@@ -222,7 +220,7 @@ func deleteAzureStackPublicRecords(ctx context.Context, o *ClusterUninstaller) e
 
 	recordsClient := azurestackdns.NewRecordSetsClientWithBaseURI(o.Environment.ResourceManagerEndpoint, o.SubscriptionID)
 	recordsClient.Authorizer = o.Authorizer
-	clusterName := o.ClusterName
+	clusterName := o.InfraID
 
```

Comment:
Should change the variable name now that this is no longer the cluster name. Not sure if we even need a variable? Could just use `o.InfraID` driectly.

Reply:
Yeah would be better actually